### PR TITLE
Ticket-NLM1736

### PR DESCRIPTION
**Pull request by cntr-tbeckford**

### CODE REVIEW

**Comment by cntr-tbeckford** [comment]
PR- To push changes for Ticket-NLM-1736

File: cypress/integration/AW_DEC_Enroll.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Feature: AWCO Enrollment(EOL)
```
Comment (antchez) [inline cypress/integration/AW_DEC_Enroll.feature:1]: This Feature is related to AW and DEC. Can we change the name to "AW and DEC Enrollment (EOL)" or something along those lines?

File: cypress/support/commands.js [hunk 2]
```diff
@@ -61,8 +61,10 @@ Cypress.Commands.add('sendForm', () => {
     cy.get('#submit').click();
 })
 
-Cypress.Commands.add('submitForm', () => {
-    cy.get('#doit').click();
+//Updated: Some buttons IDs are #doit but all buttons have type=submit
```
Comment (antchez) [inline cypress/support/commands.js:64]: Thank you!

**Review by antchez — CHANGES_REQUESTED** [review]
This looks like a mix between NLM-1736, 1735, and 1738. Once we change the title of the Feature file for enrollment and remove the new fixture file, we should be good to go.

**Pull request closed without merging**